### PR TITLE
release: Preparing CLN v24.02

### PR DESCRIPTION
**Pull request by cdecker**
- release: Update the CHANGELOG.md
- release: Bump pyln package versions

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,114 @@ All notable changes to this project will be documented in this file.
...
+ - lightningd: Speed up blocksync by not parsing unused parts of the transactions ([#6984])
+ - core: Processing blocks should now be faster ([#6984])
+ - Config: `experimental-anchors` now does nothing (it's enabled by default). ([#6785])
+ - reckless installs python plugins in virtual environments ([#7018])
+ - Protocol: `option_gossip_queries` is now required (advertized by all but 16 nodes) ([#6864])
+ - Protocol: `option_gossip_queries` is now required (advertized by all but 11 nodes) ([#6864])
+ - Protocol: `option_data_loss_protect` is now required (advertized by all but 11 nodes) ([#6864])
+ - core: Processing blocks should now be faster ([#6983])
+ - Enable optimizations for libwally/libsecp256k1-zkp ([#6983])
+ - Update libwally to 1.0.0 ([#6983])
```
Comment (rustyrussell) [inline CHANGELOG.md:45]: We actually are at libwally 1.2.0 (plus large-block fix).

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,114 @@ All notable changes to this project will be documented in this file.
...
+ - Protocol: we no longer ratelimit gossip messages by channel, making our code far simpler. ([#6941])
+ - Config: `disable-ip-discovery` (deprecated in v23.02): use `announce-addr-discovered` ([#6936])
+ - wallet: removal of p2sh-segwit addresses; newaddr won't issue them, we won't watch them for new funds (deprecated in *23.02*) ([#6936])
+ - JSON-RPC: `invoice`, `sendonion`, `sendpay`, `pay`, `keysend`, `fetchinvoice`, `sendinvoice`: `msatoshi` argument (deprecated 0.12.0). Use `amount_msat`. ([#6936])
+
+
+### Fixed
+
+ - JSON-RPC: `close` with `destination` works even if prior `destination` was rejected. ([#7072])
+ - `channel_type` reflects option_zeroconf if explicitly negotiated. ([#6864])
```
Comment (rustyrussell) [inline CHANGELOG.md:71]: `JSON-RPC` prefix missing (my bad!)

File: CHANGELOG.md [hunk 3]
```diff
@@ -2759,6 +2867,7 @@ There predate the BOLT specifications, and are only of vague historic interest:
 6. [0.5.1] - 2016-10-21
 7. [0.5.2] - 2016-11-21: "Bitcoin Savings & Trust Daily Interest II"
 
+[24.02rc1]: https://github.com/ElementsProject/lightning/releases/tag/v24.02
```
Comment (rustyrussell) [inline CHANGELOG.md:2870]: Tag is actually 24.02rc2, and needs to change for release anyway?

**Review by rustyrussell — COMMENTED** [review]
Minor textual cleanups....

**Comment by cdecker** [comment]
Thanks @rustyrussell, I amended the CHANGELOG accordingly :+1:

**Pull request merged**